backend/src/data/models/user/user.model.ts

```diff
@@ -28,6 +28,22 @@ class User extends Abstract {
           to: `${DbTableName.GROUPS}.id`,
         },
       },
+      interviewee: {
+        relation: Model.HasManyRelation,
+        modelClass: Interview,
```

Comment:
isn't it cycle dependency now?
user model imports interview model
interview imports user model

Reply:
removed this part, done